.github/workflows/tests.yml

```diff
@@ -0,0 +1,27 @@
+name: Python package
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5]
```

Comment:
Can we use the same pattern from this comment: https://github.com/appsembler/edx-organizations/pull/22#pullrequestreview-904530722